src/apps/myaccount/submodules/billing/billing.js

```diff
@@ -40,28 +40,86 @@ define(function(require) {
 							callback(null, {});
 						}
 					});
+				},
+				accountToken: function(callback) {
+					self.callApi({
+						resource: 'billing.getToken',
+						data: {
+							accountId: self.accountId
+						},
+						success: function(data, status) {
+							callback(null, data.data);
+						}
+					});
 				}
 			}, function(err, results) {
 				self.billingFormatData(results, function(results) {
 					var billingTemplate = $(self.getTemplate({
-						name: 'layout',
-						data: results,
-						submodule: 'billing'
-					}));
+							name: 'layout',
+							data: results,
+							submodule: 'billing'
+						})),
+						setCardHeader = function(creditCard, button) {
+							var cardType = _.toLower(creditCard.card_type) === 'american express'
```

Comment:
not really needed to use lodash#toLower, native js .toLowerCase() will provide bette rperformace and same result